server/src/main/java/org/apache/iotdb/db/engine/cache/LRULinkedHashMap.java

```diff
@@ -55,7 +55,7 @@
   protected long averageSize = 0;
 
   public LRULinkedHashMap(long maxMemory) {
-    this.linkedHashMap = new LinkedHashMap<>(INITIAL_CAPACITY, LOAD_FACTOR_MAP);
```

Comment:
Hi, 
I am not sure whether it is enough by just enabling `accessOrder = true`.

If you read the `put()` method, you will find it removes elements at the head of the set, rather than the tail...

Do we need to use `removeEldestEntry ()` method instead? @JackieTien97

Reply:
I read the `put()` method, and I think it is ok to just enable `accessOrder = true`.
1、At line 63 of the current file, the removal of elements is realized outside of the LinkedHashMap.
2、Everytimes after an element is accessed, the element is put at the tail, so the head is the eldest one.
3、As the default value returned by `removeEldestEntry()` method is false, the `put()` method of LinkedHashMap does not remove any elements.

Reply:
I read the `put()` method, and I think it is ok to just enable `accessOrder = true`.
1、At line 63 of the current file, the removal of elements is realized outside of the LinkedHashMap.
2、Everytimes after an element is accessed (including the `put()` method and the `get()` method), the element is put at the tail, so the head is the eldest one.
3、As the default value returned by `removeEldestEntry()` method is false, the `put()` method of LinkedHashMap does not remove any elements.

Reply:
1. but why not use `removeEldestEntry `? 
Can calculating the size in `put` bring any benefit ?

2. About added UT:

The UT does not take effect. Even you do not modify `LRULinkedHashMap.java` and only add your UT codes, the UT can also pass, which means the UT does not cover the incorrection of the original `LRULinkedHashMap.java`.

Reply:
1、extend `removeEldestEntry()` method
2、improve UT